### PR TITLE
Update the file patching documentation post A3 v1.50

### PR DESCRIPTION
**Pull request by esteldunedain**
Learned this the hard way. I hope I got it right,

### CODE REVIEW

**Comment by SilentSpike** [comment]
:+1:

File: wiki/development/setting-up-the-development-environment.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ To create a complete build of ACE that you can use without the source files, run
 
 File Patching allows you to change the files in an addon while the game is running, requiring only a restart of the mission. This makes it great for debugging, as it cuts down the time required between tests. Note that this only works with PBOs created using MakePBO, as outlined in [Creating a Test Build](#creating-a-test-build).
 
-### 7.1 Enabling File Patching
+### 7.1 Disabling CBA funcion caching
```
Comment (bux578) [inline wiki/development/setting-up-the-development-environment.md]: function

File: wiki/development/setting-up-the-development-environment.md [hunk 2]
```diff
@@ -103,7 +103,11 @@ class CfgSettings {
 };
 ```
 
-### 7.2 Restrictions of File Patching
+### 7.2 Running Arma 3 with File Patching enabled
```
Comment (bux578) [inline wiki/development/setting-up-the-development-environment.md]: File Patching vs file patching

Comment (esteldunedain) [inline wiki/development/setting-up-the-development-environment.md]: Do you prefer uppercase or lowercase?

**Pull request merged**